tests/by_image/base-notebook/test_container_options.py

```diff
@@ -47,8 +47,7 @@ def test_nb_user_change(container: TrackedContainer) -> None:
     )
     command = f'stat -c "%F %U %G" /home/{nb_user}/.jupyter'
     expected_output = f"directory {nb_user} users"
-    exec_result = running_container.exec_run(command, workdir=f"/home/{nb_user}")
-    output = exec_result.output.decode().strip("\n")
```

Comment:
There were some `.strin("\n")` - I don't think they are actually needed, so let's try to remove them